build-system/tasks/extension-generator/jscodeshift/insert-extension-bundles-config.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const arrayAssignmentIdName = 'extensionBundles';
+
+/**
+ * Inserts an extension entry into bundles.config.js
+ *
+ * ```
+ *    const extensionBundles = [
+ *      ...,
+ *   +  {
```

Comment:
Can we just maintain this as a JSON file? We could skip the weird transform then, and just `JSON.parse`, mutate, `JSON.stringify` (and prettier).

Reply:
You're right, now that `type` is gone there aren't any more JS references. JSON is a lot simpler.